crates/database/Cargo.toml

```diff
@@ -12,6 +12,7 @@ const_format = "0.2"
 futures = { workspace = true }
 hex = { workspace = true }
 sqlx = { workspace = true }
+strum = { version = "0.25", features = ["derive"] }
```

Comment:
We also have `strum` in the workspace `cargo.toml`.

Reply:
fixed